inbm/telemetry-agent/telemetry/telemetry_handling.py

```diff
@@ -303,7 +303,7 @@ def get_query_related_info(option: str, info: Dict) -> Dict:
                 del info['values'][elem]
     elif option == "version":
         info = {}
-        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()        
+        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
```

Comment:
<h3>$\color{#FFD700}Coding Style: CodeConsistency$</h3>

Ensure that the removal of trailing whitespace is consistent across the entire codebase to maintain uniformity.